api.html

```diff
@@ -16,7 +16,7 @@ <h1 class="hero-main-header">IDR API</h1>
                     The Image Data Resource (IDR) provides a JSON-based API for accessing all datasets,
                     thumbnails and metadata. This API is a combination of the standard
                     <a href="https://www.openmicroscopy.org/site/support/omero/developers/Web.html">OMERO.web API</a>
-                    and newly developed plugins like <a href="https://github.com/ome/omero-mapr/tree/master/docs">OMERO.mapr</a>.
+                    and newly developed <a href="https://github.com/ome/omero_search_engine/">search engine</a>.
```

Comment:
Probably worth linking to https://idr.openmicroscopy.org/searchengine/apidocs/ somewhere?